e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -221,7 +221,17 @@ function visitDashboardById(dashboard_id, config) {
   }).then(({ status, body: { dashcards, tabs } }) => {
     const dashboardAlias = "getDashboard" + dashboard_id;
 
-    cy.intercept("GET", `/api/dashboard/${dashboard_id}*`).as(dashboardAlias);
+    cy.intercept(
+      {
+        method: "GET",
+        url: `/api/dashboard/${dashboard_id}*`,
+      },
+      req => {
+        req.reply(res => {
+          res.body.last_used_param_values = {}; // remove param values to prevent random redirects caused by cached redirect
+        });
+      },
```

Comment:
I think this may have broken some other tests? 🤔

Reply:
did it? I was wondering how do we even match this intercept. it does not seem like we wait for it in any tests, we just match this request for no apparent reason. I had to create that waitAlias utility to match this intercept